frontend/web/src/app/model/order.ts

```diff
@@ -0,0 +1,18 @@
+export class Customer {
+
+  public id: number;
+  public customer: number;
+
+  constructor(values: Object = {}) {
+    if (!values)
+      return null;
+    Object.assign(this, values);
+  }
+
+  static create(json: any) {
+    if (json)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') if statements must be braced [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS121)